### ISSUE TITLE
Change default branch to master

### ISSUE THREAD

**Issue by conao3** [wanderlust/apel#8]
Hi.
Now this project's default branch set `apel-wl` but it is not common.
Are there any reasons to change the default branch?
It's hard to remember the rules for each project, so could you change it to a generic branch name?

**Comment by ikazuhiro** [comment]
Because this repository is unofficial and clone of [original](http://git.chise.org/elisp/apel/) except `apel-wl` branch.  We could change developmental branch name but I think the name should be non-existing one in original repository.

**Comment by conao3** [comment]
If you think this project is *unofficial*, but this project is curated by MELPA and most users think this ploject is upstream and official APEL.

quoted from [MELPA homepage](https://melpa.org/#/)
> Up-to-date packages built on our servers from upstream source
    Installable in any Emacs with 'package.el' - no local version-control tools needed
    **Curated - no obsolete, renamed, forked or randomly hacked packages**
    Comprehensive - more packages than any other archive
    Automatic updates - new commits result in new packages
    Extensible - contribute recipes via github, and we'll build the packages

And from [MELPA CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org#what-to-know-before-opening-a-pull-request)
> **Official repository**
    Packages should be built from the official package repository. Forks of the official repository will not be accepted except in extreme circumstances.

**Comment by ikazuhiro** [comment]
> If you think this project is unofficial

It is not my thought, just a fact.  We can now announce this project is official APEL-LB, but not APEL.

> but this project is curated by MELPA and most users think this ploject is upstream and official APEL.

I think only authors (copyright owners) can decide what is official, neither MELPA nor most users.  Additionally, I guess most users have no interest in whether this is official or not.

> > **Official repository**
> > Packages should be built from the official package repository. Forks of the official repository will not be accepted except in extreme circumstances.

I guess APEL, FLIM, SEMI and Wanderlust are exactly the case *in extreme circumstances* (original ones are not usable or lost).  Have you ever confirmed that MELPA's maintainer really think this project is upstream and official?

**Comment by conao3** [comment]
> I think only authors (copyright owners) can decide what is official, neither MELPA nor most users. Additionally, I guess most users have no interest in whether this is official or not.

> I guess APEL, FLIM, SEMI and Wanderlust are exactly the case in extreme circumstances (original ones are not usable or lost). Have you ever confirmed that MELPA's maintainer really think this project is upstream and official?

No. These are MELPA requirements and are mandatory for submitting recipes.
If you call this project `apel-lb`, it should be registered with MELPA as `apel-lb`.
The project is already registered as `apel` with MELPA. In fairness, the project should be considered as apel upstream.

But, if you think welcoming to merge `semi`, `flim`, `apel` ploject into `wanderlust`, I can ignore this issue.

**Comment by ikazuhiro** [comment]
> > I guess APEL, FLIM, SEMI and Wanderlust are exactly the case in extreme circumstances (original ones are not usable or lost). Have you ever confirmed that MELPA's maintainer really think this project is upstream and official?
> 
> No. These are MELPA requirements and are mandatory for submitting recipes.

1. As you quoted, MELPA does not claim that packages MUST be built from the official package repository.
2. I interpret "Forks of the official repository will not be accepted except in extreme circumstances." as that unofficial repository may be accepted.
3. Erik (who worked for registering Wanderlust and related project) had sent a PR for APEL and withdrawn once because ["there is an official apel git repo"](https://github.com/melpa/melpa/pull/1663#issuecomment-40912141).  6 month later, [APEL had been added into MELPA](https://github.com/melpa/melpa/pull/2132).  If MEPLA's maintainer remembered it, he should have known that this repository has been unofficial.
4. For example, [MU-CITE](https://melpa.org/#/mu-cite)'s repository is apparently unofficial, but [it doesn't seem to be checked](https://github.com/melpa/melpa/pull/3547).  I doubt MELPA's curation reliability especially for old packages's officiality.
5. So I think to be registered in MELPA does not always means that MELPA's maintainer think the package's repository is upstream and official.  In APEL's case, that is very doubtful.

> But, if you think welcoming to merge `semi`, `flim`, `apel` ploject into `wanderlust`, I can ignore this issue.

I'll welcome when such integrated package is available.  Whether it should be merged with Wanderlust whould depend on packages's modularity.  If it is easily usable from other package, it would be better distributed as independent package.  If it tightly connected to Wanderlust, it should be merged into Wanderlust.
